cs-vmware/src/main/java/io/cloudslang/content/vmware/utils/InputUtils.java

```diff
@@ -138,4 +139,10 @@ public static void checkMutuallyExclusiveInputs(final String input1, final Strin
             throw new IllegalArgumentException(exceptionMessage);
         }
     }
+
+    public static void checkOptionalMutuallyExclusiveInputs(final String input1, final String input2, final String exceptionMessage) {
+        if (isNotBlank(input1) && isNotBlank(input2)) {
```

Comment:
Ob, but then the description of the inputs is not clear, 
```
     * @param virtualMachineName    The name of the virtual machine for which the override will be created. This input
     *                              is mutually exclusive with virtualMachineId.
     * @param virtualMachineId      The id of the virtual machine for which the override will be created. This input is
     *                              mutually exclusive with virtualMachineName.
```
at least add that they are optional